README.md

```diff
@@ -54,28 +54,30 @@
 
 ## About The Project
 
-[![Product Name Screen Shot][product-screenshot]](https://example.com)
+![Product Name Screen Shot][product-screenshot]
 
 Robusta KRR (Kubernetes Resource Recommender) is an extensible CLI tool designed to help optimize resource allocation in Kubernetes clusters. It gathers pod usage data from Prometheus, and applies customizable strategies to calculate CPU and memory requests and limits recommendations. This helps you get the most out of your cluster resources and improve overall efficiency.
 
 ### Features
 
--   Extensible Strategies: Easily create and use your own strategies for calculating resource recommendations.
--   Custom Formatters: Write custom formatters to present the results in your preferred format.
+-   No Agent Required: Robusta KRR is a CLI tool that runs on your local machine, and does not require any agents to be installed on your cluster.
 -   Prometheus Integration: Gather resource usage data using built-in Prometheus queries, with support for custom queries coming soon.
+-   Custom Formatters: Write custom formatters to present the results in your preferred format.
```

Comment:
I would still remove customer formatters. Just explain the default formatters and why they're good.